ohif-configurable/hp-extension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@radicalimaging/hp-extension",
-  "version": "3.9.1",
+  "version": "3.9.3",
```

Comment:
there is a released version 3.9.4, So the next value of this should be at least 3.9.5.
The value here will depend if there is CI routine which updates it at end